test/integration/local/test_python_service.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+import subprocess
+import sys
+import time
+
+import pytest
+
+import requests
+
+PING_URL = 'http://localhost:8080/ping'
+INVOCATIONS_URL = 'http://localhost:8080/invocations'
+
+
+@pytest.fixture(scope='session', autouse=True)
+def volume():
+    try:
+        model_dir = os.path.abspath('test/resources/models')
+        subprocess.check_call(
+            'docker volume create --name model_volume --opt type=none '
+            '--opt device={} --opt o=bind'.format(model_dir).split())
+        yield model_dir
+    finally:
+        subprocess.check_call('docker volume rm model_volume'.split())
+
+
+@pytest.fixture(scope='module', autouse=True, params=['1.11', '1.12'])
+def container(request):
+    try:
+        command = (
+            'docker run --name sagemaker-tensorflow-serving-test -p 8080:8080'
+            ' --mount type=volume,source=model_volume,target=/opt/ml/model,readonly'
+            ' -e SAGEMAKER_TFS_DEFAULT_MODEL_NAME=half_plus_three'
+            ' -e SAGEMAKER_TFS_NGINX_LOGLEVEL=info'
+            ' -e SAGEMAKER_BIND_TO_PORT=8080'
+            ' -e SAGEMAKER_SAFE_PORT_RANGE=9000-9999'
+            ' -e "GUNICORN_PORT=5000'
```

Comment:
inside sage_port_range. also, users shouldn't need to set this... the container should just pick a port within the safe port range and use it.